src/components/Notifications/Notification.js

```diff
@@ -21,9 +21,12 @@ function Notification({ notification }) {
       {text}
 
       <div className="mt-2">
-        <button className="text-blue-600 font-bold mr-2" onClick={() => dispatch(actions.close(id))}>
+        <button className="text-blue-600 font-bold mr-2" onClick={() => dispatch(actions.close(notification.id))}>
           Dismiss
         </button>
+        <button className="text-blue-600 font-bold mr-2" onClick={() => dispatch(actions.blacklistAndClose(notification))}>
```

Comment:
For now, I'd use `text-blue-300` for this action.

https://tailwindcss.com/docs/customizing-colors#default-color-palette